projects/plugins/crm/includes/ZeroBSCRM.MetaBoxes3.Events.php

```diff
@@ -726,51 +726,30 @@ function zeroBSCRM_task_ui_for_co($taskObject = array()){
 
 }
 
-
-#} the date picker UI
-function zeroBSCRM_task_ui_date($taskObject = array()){
-
-    $html = "<div class='no-task-date'><i class='ui icon calendar outline'></i> ". __('Date','zero-bs-crm') ." </div>";
```

Comment:
The variable was overridden on line 769, so there's no need for us to keep it around.
As far as I can see, then it must have been used when together with an early bail out `return` so we'd show it as a default value?